### PR TITLE
feat(image): add image preview description template

### PR DESCRIPTION
**Pull request by denyssalon**
### Feature Requests
This PR extends image component with custom image preview description template 
Screenshot 2024-10-07 at 13 34 34

### CODE REVIEW

File: src/app/components/image/image.interface.ts [hunk 1]
```diff
@@ -29,4 +29,9 @@ export interface ImageTemplates {
      * Custom template of closeicon.
      */
     closeicon(): TemplateRef<any>;
+
+    /**
+     * Custom template of imagedescription.
```
Comment (SoyDiego) [inline src/app/components/image/image.interface.ts:34]: space here? `image description`

Comment (denyssalon) [reply src/app/components/image/image.interface.ts:34]: @SoyDiego It is a name of the template which is imagedescription (without space).
I made it the same way as comments above (closeicon, rotaterighticon etc). Do you still want me to put a space there?

Comment (SoyDiego) [reply src/app/components/image/image.interface.ts:34]: oh ok, sorry, thanks for re check!
Now we should wait if PrimeNG Team approve it!

**Review by SoyDiego — APPROVED** [review]

**Comment by denyssalon** [comment]
Hello @SoyDiego. Do you think this has a chance to be merged someday?

**Comment by SoyDiego** [comment]
> Hello @SoyDiego. Do you think this has a chance to be merged someday?

Sorry, i'm not part of PrimeNG Team, only a contributor like you :).
They will do it for sure, but sometimes we need to wait because they are working in the v18

**Comment by mertsincan** [comment]
Hi @denyssalon,

Thanks a lot for your contribution! Could you please create a PR for v18 as well?

**Comment by mertsincan** [comment]
Needs issue and it has conflicts. Please create a new PR. 

Thanks a lot!

**Pull request closed without merging**